src/pages/LoginRegister/LoginPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useContext } from "react";
+import "./LoginPage.css";
+import { useNavigate } from "react-router-dom";
+import LoginForm from "./LoginForm";
+import AuthContext from "../../context/AuthProvider";
+
+export default function LoginPage() {
+  const navigate = useNavigate();
+  const [errors] = useState("");
+  /*testing array
+  const adminUser = {
+    email: "admin@123",
+    username: "admin",
+    password: "12ABqwer",
+  };*/
+  const { userDetails } = useContext(AuthContext);
+
+  const [isSuccess] = useState(false);
+
+  const Login = (details) => {
+    console.log(details);
```

Comment:
Please remove the console.logs from the code, they are security vulnerabilities.
```suggestion
```